pandas/tests/scalar/period/test_period.py

```diff
@@ -67,7 +67,7 @@ def test_construction(self):
         assert i1 == i3
 
         i4 = Period("2005", freq="M")
-        i5 = Period("2005", freq="m")
+        i5 = Period("2005", freq="me")
```

Comment:
what is "me"?

Reply:
month end

unrelated to this issue, but `'m'` (lowercase) is interpreted as `'M'` (uppercase):
```python
In [7]: Period('2020-01', 'm').freq
Out[7]: <MonthEnd>
```
and I presume she's staying consistent with the current lower-casedness

Reply:
might not be relevant to this PR, but i find this confusing.  my first guess would have been that "m" is "minute"

Reply:
sure, I've opened https://github.com/pandas-dev/pandas/issues/54105